hack/make-rules/release.sh

```diff
@@ -0,0 +1,196 @@
+  "keadm"
+)
+
+function release() {
+  local -a targets=()
+  local VERSION=""
+  local ARCH="amd64"
+
+  for arg in "$@"; do
+    if [[ "${arg}" == GOARM7 ]]; then
+      ARCH="arm"
+    elif [[ "${arg}" == GOARM8 ]]; then
+      ARCH="arm64"
+    else
+      targets+=("${arg}")
+    fi
+  done
+
+  if [[ ${#targets[@]} -eq 0 ]]; then
+    targets=("${ALL_RELEASE_TARGETS[@]}")
+  fi
+
+  kubeedge::version::get_version_info
+  VERSION=${GIT_VERSION}
+
+  for bin in ${targets[@]}; do
+    case "${bin}" in
+      "edgesite")
+        if [ "${ARCH}" == "amd64" ]; then
+          hack/make-rules/build.sh edgesite-server edgesite-agent
+        else
+          hack/make-rules/crossbuild.sh edgesite-server edgesite-agent ${GOARM}
+        fi
+
+        build_edgesite_release $VERSION $ARCH
+        ;;
+      "keadm")
+        if [ "${ARCH}" == "amd64" ]; then
+          hack/make-rules/build.sh keadm
+        else
+          hack/make-rules/crossbuild.sh keadm ${GOARM}
+        fi
+
+        build_keadm_release $VERSION $ARCH
+        ;;
+      "kubeedge")
+        if [ "${ARCH}" == "amd64" ]; then
+          hack/make-rules/build.sh cloudcore admission edgecore csidriver iptablesmanager
+        else
+          hack/make-rules/crossbuild.sh cloudcore admission edgecore csidriver iptablesmanager ${GOARM}
+        fi
+
+        build_kubeedge_release $VERSION $ARCH
+        ;;
+      *)
+        echo "not supported release:" $bin "only supported:" ${ALL_RELEASE_TARGETS[@]}
+        exit 1
+    esac
+  done
+}
+
+function build_kubeedge_release() {
+  local VERSION=""
+  local ARCH="amd64"
+
+  for arg in "$@"; do
+    if [[ "${arg}" == v* ]]; then
+      VERSION="${arg}"
+    elif [[ "${arg}" == arm* ]]; then
+      ARCH="${arg}"
+    fi
+  done
+
+  echo "building kubeedge release:" ${VERSION} "ARCH:"${ARCH}
+
+  mkdir -p _output/local/kubeedge-${VERSION}-linux-${ARCH}/cloud
+  mkdir -p _output/local/kubeedge-${VERSION}-linux-${ARCH}/cloud/admission
+  mkdir -p _output/local/kubeedge-${VERSION}-linux-${ARCH}/cloud/cloudcore
+  mkdir -p _output/local/kubeedge-${VERSION}-linux-${ARCH}/cloud/csidriver
+  mkdir -p _output/local/kubeedge-${VERSION}-linux-${ARCH}/cloud/iptablesmanager
+  mkdir -p _output/local/kubeedge-${VERSION}-linux-${ARCH}/edge
+
+  echo ${VERSION} > _output/local/kubeedge-${VERSION}-linux-${ARCH}/version
+  cp _output/local/bin/admission _output/local/kubeedge-${VERSION}-linux-${ARCH}/cloud/admission
+  cp _output/local/bin/cloudcore _output/local/kubeedge-${VERSION}-linux-${ARCH}/cloud/cloudcore
+  cp _output/local/bin/csidriver _output/local/kubeedge-${VERSION}-linux-${ARCH}/cloud/csidriver
+  cp _output/local/bin/iptablesmanager _output/local/kubeedge-${VERSION}-linux-${ARCH}/cloud/iptablesmanager
+
+  cp _output/local/bin/edgecore _output/local/kubeedge-${VERSION}-linux-${ARCH}/edge
+
+  cd _output/local
+  tar -czvf ${KUBEEDGE_ROOT}/_output/local/kubeedge-${VERSION}-linux-${ARCH}.tar.gz kubeedge-${VERSION}-linux-${ARCH}/
```

Comment:
Just have a try, run as expected! How about create a `release` dir under `local`, and also create a subdir named as version like `v1.9.0`, then move all the release package under it?